app.rb

```diff
@@ -65,6 +61,18 @@ def createperson
     end
   end
 
+  def no_parent_permission(age, name)
+    student = Student.new(age, name, false, nil) # Provide all four arguments
+    @peopl.push(student)
+    puts 'Person created successfully'
+  end
+
+  def having_parent_permission(age, name)
+    student = Student.new(age, name, true, nil)
+    @people.push(student)
+    puts 'Person created successfully'
+  end
```

Comment:
- [ ] Please 🙏🏻 follow the ruby best practices, as you can see your functionality inside the two functions is similar except the parameters `true` and `false`. So please use the one function and re-use it again and again, which will help you to achieve more drier code.